src/cljc/athens/common_events.cljc

```diff
@@ -307,6 +307,34 @@
                      :new-uid new-uid}}))
 
 
+(defn build-unlinked-references-link
+  "Builds `:datascript/unlinked-references-link` event with:
+  - `uid`:  `:block/uid` of the block with unlinked reference
+  - `string `: content of the block
+  - `title  `: title of the page"
+  [last-tx uid string title]
+  (let [event-id (gen-event-id)]
+    {:event/id      event-id
+     :event/last-tx last-tx
+     :event/type    :datascript/unlinked-references-link
+     :event/args    {:uid    uid
+                     :string string
+                     :title  title}}))
+
+
+(defn build-unlinked-references-link-all
+  "Builds `:datascript/unlinked-references-link` event with:
+  - `unlinked-refs`:  vector of blocks with unlinked refs
```

Comment:
"vector of blocks" as in whole entities?

Reply:
> whole entities
Can you clarify?

Reply:
I think here Alex means are we sending _whole individual block data_ or just block-uids?

for e.g _whole individual block data_ looks like this: 
```
{:db/id              -3
  :block/uid       "some-block-uid"
  :block/string    "some block text"
  :block/order     1
   :block/children []}
```

Reply:
I checked and it is the case that we are sending whole block data, this is not the right way to do this because block data will also contain `:db/id` which we don't want to send to the remote. Can we calculate unlinked refs on the resolver side? What basic data do we need to know so that unlinked refs can be calculated on the resolver side?

Reply:
Updated